src/main/java/Chelle/Chelle.java

```diff
@@ -0,0 +1,174 @@
+package Chelle;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import ChelleCommands.*;
+import ChelleExceptions.*;
+
+public class Chelle {
+    private static final String FILE_PATH = "./ChelleTasks.txt";
+
+
+    public static void main(String[] args) {
+        // Load tasks from the hard disk when the chatbot starts up
+        ArrayList<Task> tasks = SaveTasks.loadTasksFromFile();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Chelle.\nI like to talkity talkity talk!");
+
+        while (true) {
+            System.out.print("You: ");
+            String userInput = scanner.nextLine();
+            CommandType userCommand;
+            try {
+                userCommand = CommandType.valueOf((userInput.split(" ")[0]).toUpperCase());
+            } catch (IllegalArgumentException e) {
+                System.out.println("Chelle: Invalid command. Please start your command with one of the following commands: \n" +
+                        "'list', 'mark', 'unmark', 'todo', 'deadline', 'event', or 'delete'.");
+                continue;
+            }
+
+            switch (userCommand) {
+            case BYE:
+                SaveTasks.saveTasksToFile(tasks);
+                System.out.println("Chelle: Bye! Hope to see you again soon!");
+                scanner.close();
+                return;
+
+            case LIST:
+                System.out.println("Chelle: ");
+                displayTasks(tasks);
+                break;
+
+            case MARK:
+            case UNMARK:
+            case TODO:
+            case DEADLINE:
+            case EVENT:
+            case DELETE:
```

Comment:
add //Fallthrough statements for empty catch blocks, see the coding standards for more info